packages/synapse-react-client/src/lib/containers/UserSearchBoxV2.tsx

```diff
@@ -73,6 +73,7 @@ const customSelectComponents: Partial<
           userGroupHeader={data.header}
           disableHref={true}
           showFullName={true}
+          showCardOnHover={false}
```

Comment:
When the mouse hovered over the username in the search results, the search results would disappear!